frontend/src/components/CourseSearchBar/CourseSearchBar.tsx

```diff
@@ -62,7 +115,7 @@ const CourseSearchBar = ({ onSelectCallback, style }: Props) => {
       onSearch={handleSearch}
       onSelect={handleSelect}
       notFoundContent={isLoading && value && <Spin size="small" />}
-      style={{ width: '30rem', ...style }}
+      style={{ width: '55ch', ...style }}
```

Comment:
knitpick: mayb back to rem just for consistency and its more understandable 🥺

Reply:
(non urgent)

Reply:
Can do in future fix. Added to (mental) backlog